lib/surface_site_web/live/getting_started.ex

```diff
@@ -75,6 +75,86 @@ defmodule SurfaceSiteWeb.GettingStarted do
                 ]
                 ```
 
+                ## Integration
+
+                We will assume that you already have LiveView already implemented in your project. (See the [`Phoenix LiveView` documentation](https://hexdocs.pm/phoenix_live_view/installation.html#content) for greater detail on how to accomplish this).
+
+                A simple example migration from `Phoenix.LiveView` consists of the following steps.
+
+                ### Ensure you have a LiveView available
+
+                Ensure that you have `LiveView` configured correctly in your project by implementing a `live` route directed to a `LiveView`.
+
+                ```elixir
+                live "/", ExampleLive, :index
+                ```
+
+                Surface offers drop-in replacement components for `Phoenix.LiveView` (`Surface.LiveView`), `Phoenix.Component` (`Surface.Component`), and `Phoenix.LiveComponent` (`Surface.LiveComponent`).
+
+                Let's create our first `Surface.Component` and render it in our `Surface.LiveView`.
+
+
+                ### Create your first Surface.Component
+
+                Create a module for your component. Similarly to a `Phoenix.LiveView` component, `Surface` supports co-locating templates within a sigil (`~F` rather than `~L/~H`) or a template file with the same file name and the extension (`.sface` rather than `.leex/.heex`).
+
+
+                ```elixir
+                defmodule MyAppWeb.Components.ExampleComponent do
+                  use Surface.Component
+
+                  slot default, required: true
+
+                  def render(assigns) do
+                    ~F"\""
+                    <h1><#slot /></h1>
+                    "\""
+                  end
+
+                  # Or omit the render function and create a file
+                  # with the same filename as the component but with
+                  # an `.sface` extension
+
+                end
+                ```
+
+                ### Converting a Phoenix.LiveView to Surface.LiveView
+
+                Change the macro in your `LiveView` from `Phoenix.LiveView` (or `YourApp, :live_view`) and the render sigil from `~H/~L` or your template extension from `.heex`/`.leex` to `.sface`.
```

Comment:
In my projects, I do not modify the `live_view` macro. Instead, I prefer to create a new `surface_live_view` macro. The main reason is that I do not want to confuse newcomers who don't know Surface. I prefer to be explicit. That being said, I believe I do not use the old macro any more, so it's just death code :sweat_smile: Not sure what the best strategy.

Reply:
I appreciate what you're suggesting. The approach I took was really to emphasize the bare minimum to demonstrate a replacement of both the `Surface.LiveView` and `Surface.Component` to prove the functionality. 
I have started discussing your suggested approach in [an issue on the main repo](https://github.com/surface-ui/surface/issues/487) and am working on another piece of documentation to discuss this very issue. I do feel that on this particular section of the documentation, it would be better to stick with the very basics.